res/TensorFlowLiteRecipes/Pad_002/test.rule

```diff
@@ -0,0 +1,6 @@
+# To check if dynamic dimension properly infered
+
+RULE    "VERIFY_FILE_FORMAT"      $(verify_file_format) '=' 1
+
+RULE    "PAD_EXIST"               $(op_count PAD) '=' 1
+RULE    "PAD_SHAPE"               $(tensor_shape ofm) '=' -1572
```

Comment:
`-1 5 7 2` is more readable than `-1572`. My suggestion is to format it as `$(tensor_shape ofm) = '-1 5 7 2'`, with the appropriate adjustments in the `DumpTensorShape::run` function in `dump.cpp`.

Reply:
> `-1 5 7 2` is more readable than `-1572`.

I agree. 
Actually, `DumpTensorShape::run` isn't problem for this.
In `dredd-rule-lib/rule-lib.sh`,
```
ACTUAL=`init_error_log ; \
          ${INSPECT_PROG_PATH} --tensor_shape ${COMPILED_FILE} | \
          awk -v tensor_name="$1" '{ if ($1 == tensor_name) print $2}'`
```

this line has problem.  To properly fix rule with blank, we have to add `for-loop` in `.sh` which seems too much for me.

Reply:
How about `-1,5,7,2`?